bin/csv2avro

```diff
@@ -48,11 +48,12 @@ begin
 
   CSV2Avro.new(options).convert
 rescue OptionParser::MissingArgument => ex
-  puts ex.message
-
-  puts option_parser
+  $stderr.puts ex.message
+  $stderr.puts option_parser
+  exit 2
 rescue Exception => e
-  puts 'Uh oh, something went wrong!'
-  puts e.message
-  puts e.backtrace.join("\n")
+  $stderr.puts 'Uh oh, something went wrong!'
+  $stderr.puts e.message
+  $stderr.puts e.backtrace.join("\n")
+  exit 2
```

Comment:
I'd exit with different error codes in each cases. That may help debug the issue later.